app/views/placements/pages/landing-page.md

```diff
@@ -0,0 +1,13 @@
+This service is for schools offering placements for trainee teachers and 
+Initial teacher training (ITT) providers in England 
+(accredited ITT providers and lead partners).
+
+If you are a school, use this service to record ITT placements at your organisation.
+If you are an ITT provider, use this service to find ITT school placement opportunities.
+
+##Before you start##
```

Comment:
I'm kind of surprised that this works...? 🤔 Does it generate a H2?

I was expecting to see:

```suggestion
## Before you start
```

Reply:
I've just checked locally and it _does_ generate a H2. Yay for loose standards.